python/ray/scripts/scripts.py

```diff
@@ -484,7 +576,46 @@ def start(node_ip_address, redis_address, address, redis_port, port,
             ray_params, head=True, shutdown_at_exit=block, spawn_reaper=block)
         redis_address = node.redis_address
 
-        logger.info(
+        cli_logger.newline()
```

Comment:
I think for these parts of the code, you should clearly comment/indicate why there is a redundant logging mechanism. 

This is because not everyone is familiar with the cli_logger + old_style_logging mechanisms that you have.